ddl/util/util.go

```diff
@@ -194,3 +197,25 @@ func LoadGlobalVars(ctx context.Context, sctx sessionctx.Context, varNames []str
 	}
 	return nil
 }
+
+// GetTimeZone gets the session timezone's name and offset.
```

Comment:
This is a bit more complicated, it may return a location (like 'Asia/Shanghai', 'UTC '), or it may return a fixed time zone with static offset. 
This is mainly used to store the timezone info as `model.TimeZone` to tikv and parse it to the owner worker for processing.